cmd/gitops/beta/run/cmd.go

```diff
@@ -265,7 +264,7 @@ func betaRunCommandRunE(opts *config.Options) func(*cobra.Command, []string) err
 
 		log.Actionf("Checking if GitOps Dashboard is already installed ...")
 
-		dashboardInstalled := run.IsDashboardInstalled(log, ctx, kubeClient, wego.DefaultNamespace)
+		dashboardInstalled := run.IsDashboardInstalled(log, ctx, kubeClient, "flux-system")
```

Comment:
Can we use a constant for `"flux-system"` here and in similar cases (wherever possible), similar to the defaultNamespace in `cmd/gitops/root`?

Reply:
Hm, so, shouldn't we just use the namespace from `flags.Namespace`?

Reply:
I guess it should work. Anything is fine with me but hardcoded strings. 🙂 Too easy to mistype.